admin-dev/themes/default/template/helpers/form/form.tpl

```diff
@@ -754,7 +754,7 @@
 												type="text"
 												data-hex="true"
 												{if isset($input.class)} class="{$input.class}"
-												{else}class="datepicker"{/if}
+												{else}class="datepicker" autocomplete="off"{/if}
```

Comment:
Need to be out of the if/else class related.

Reply:
No, if a class is set, the datepicker is not launched.

Reply:
You could also have the class set as "datepicker myclassname".

Reply:
or another class, autocomplete should not be forced in this case.

Reply:
We're inside the date type of the input. It's like logical.

Or, better, you could manage this option into the HelperForm controller and manage the $input.autocomplete options as it.

Reply:
@PrestaEdit there are a lot's of attribute to add from the input text 
```
class="{if isset($input.class)}{$input.class}{/if}{if $input.type == 'tags'} tagify{/if}"
{if isset($input.size)} size="{$input.size}"{/if}
{if isset($input.maxchar) && $input.maxchar} data-maxchar="{$input.maxchar|intval}"{/if}
{if isset($input.maxlength) && $input.maxlength} maxlength="{$input.maxlength|intval}"{/if}
{if isset($input.readonly) && $input.readonly} readonly="readonly"{/if}
{if isset($input.disabled) && $input.disabled} disabled="disabled"{/if}
{if isset($input.autocomplete) && !$input.autocomplete} autocomplete="off"{/if}
{if isset($input.placeholder) && $input.placeholder } placeholder="{$input.placeholder}"{/if}
```
I've tried to add this in the form.tpl but it doesn't work out of the box.
By the way, it's not the point of the issue, so feel free to open another PR to solve this, thanks Jonathan.

Reply:
I still maintain my request change regarding the needs to sort this autocomplete of the else statement, actually.

Reply:
agree, not related to the class at all, with this solution, you can end with date inputs with `datepicker` classes and no `autocomplete` attribute which is what you want to avoid there

`{if isset($input.autocomplete) && !$input.autocomplete} autocomplete="off"{/if} `

would probably be the right solution

Reply:
@NeOMakinG as I said, I've tried and it doesn't work, feel free to open a new pr for this. MY point is just to fix the bug with the date picker.

Reply:
If you want to properly fix the bug, put the autocomplete off the condition, because as you did there, it will partially fix the bug and it's not acceptable => people might added some classes with `datepicker` classes at the same time, they won't get the fix.

Reply:
Ok, i've moved the autocomplete off outside the if